Dockerfile

```
@@ -7,7 +7,7 @@ RUN apt-get --allow-releaseinfo-change update && apt-get upgrade -y && \
     build-essential \
     libz-dev \
     swig \
-    git-lfs \
+    unzip \
```

I would suggest to explicitly add `wget` here as well, although it might come with the base image already.

```suggestion
    unzip \
    wget \
```

---

Sounds good, let's do that!